.devcontainer/Dockerfile

```diff
@@ -1,21 +0,0 @@
-# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.145.0/containers/codespaces-linux/.devcontainer/base.Dockerfile
```

Comment:
We could keep this if we want to do things like install rush or other customization